addon/helpers/ehook.js

```diff
@@ -4,7 +4,7 @@
  * @see {@link https://github.com/Ticketfly/ember-hook/pull/35}
  */
 import Ember from 'ember'
-const {Helper, deprecate} = Ember
+const {Helper, deprecate, keys} = Ember
```

Comment:
They actually are alphabetized. When you sort an array of strings uppercase letters all come before lowercase. If we disagree with that then we need to update the sorting rules in `eslint-plugin-ocd`.